### PR TITLE
tutorial: introduce documentation metadata

### PR DESCRIPTION
**Pull request by rapenne-s**
This explains how to use the metadata documentation feature in a contract.

### CODE REVIEW

File: doc/manual/tuto.md [hunk 1]
```diff
@@ -186,3 +186,66 @@ be expected as we removed it earlier.
 The second part shows the contract attribute that produced the error.
 In this case it's showing that `name` should be a `Str`, and as there
 is no `optional` keyword, this attribute must be set.
+
+## Step 7: Document your Nickel file
+
+Nickel supports metadata on fields, this mean you can actually write
+documentation to explain your Nickel code. This way, your team can
+easily reuse a Nickel code without having to ask the author about a
+contract usage.
```
Comment (yannham) [inline doc/manual/tuto.md:195]: Suggestion of wording, to make this paragraph a tad lighter. Feel free to do what you want with it, including ignoring it :slightly_smiling_face: 

```suggestion
Beside contracts, you can attach various kinds of metadata to
record fields in Nickel. In particular, it is good practice to
include documentation when writing contracts.
```

File: doc/manual/tuto.md [hunk 2]
```diff
@@ -186,3 +186,66 @@ be expected as we removed it earlier.
...
+        | optional
+        | doc "List of ssh-keys in OpenSSH-specific format",
+    is-admin
+      | Bool
+      | doc "Allows the user to run some system tasks as administrator"
+      | default = false,
+    extra-groups
+      | Array Str
+      | optional
+      | doc "List of system groups the user belongs to" ,
```
Comment (yannham) [inline doc/manual/tuto.md:219]: Suggestion, just for showcasing longer documentation:

```suggestion
      | doc m%"
         List of system groups the user belongs to.
         
         # Examples
         
         ```nickel
         extra_groups = ["sudo", "sound", "nix"]
         ```
      "%,
```

**Review by yannham — APPROVED** [review]

**Comment by yannham** [comment]
It's good to go! I let you click the green button :slightly_smiling_face:

**Comment by aspiwack** [comment]
For the record, GitHub now has the option to automerge when all checks have passed (which is not without its own traps, but avoids having to actively wait for CI when you are confident).

**Pull request merged**